com.unity.shadergraph/Documentation~/Block-Node.md

```diff
@@ -0,0 +1,34 @@
+# Block Node
+
+## Description
+
+A Block is a specific type of Node for the Master Stack. A block represents a single piece of the surface (or vertex) description data used in the final shader output. Some Block nodes are always available, while some may only be available with certain pipelines.  
+
+Certain blocks are only compatible with specific Graph Settings. Blocks may become Active or Inactive based on the graph settings. 
+
+Blocks cannot be cut, copied, or pasted. 
+
+## Adding and Removing Block Nodes
+
+New Block nodes can be added to a Context in the Master Stack by hovering over empty space in the Context and pressing space bar or right clicking and selecting “Create Node”. This will bring up the Create Node menu. 
+
+This menu will contain only Block nodes valid for that context. Vertex blocks will not appear in the Create Node menu of the Fragment context. 
+
+Selecting a block node from this menu will add it to the context. 
```

Comment:
Not really selecting a block node here, can we say selecting an entry or something like that? Just in combination with the next paragraph makes it seem like you're selecting blocks and getting different behavior.